core/utils/samples.py

```diff
@@ -78,205 +78,117 @@ def add_molecule_protocol_parameters(form_data):
     return molecule_updated_list
 
 
-def analyze_input_samples(request, app_name):
-    """The function will get the samples data that user filled in the form.
-        defined_samples are the samples that either has no sample project or for
-            the sample projects that no requires additional data
-        pre_definde_samples are the ones that requires additional Information
-        For already defined samples, no action are done on them and  they are included in not_valid_samples.
-        it will return a dictionary which contains the processed samples.
+def sheet_header_to_field_name(header, field_info):
```

Comment:
Could be nice to document this function

Reply:
yes, I need to add proper docstrings to every function, you are right, I'll do it next PR :)